includes/Plugin.php

```diff
@@ -100,32 +118,33 @@ function goBack() {
             return ob_get_clean();
         }
 
-        if ( file_exists( ABSPATH . "wp-config.php" ) && is_writable( ABSPATH . "wp-config.php" ) ) {
+        if (file_exists(ABSPATH . "wp-config.php") && is_writable(ABSPATH . "wp-config.php")) {
             wp_config_delete();
-        } else if ( file_exists( dirname( ABSPATH ) . "/wp-config.php" ) && is_writable( dirname( ABSPATH ) . "/wp-config.php" ) ) {
-            wp_config_delete( '/' );
+        } else if (file_exists(dirname(ABSPATH) . "/wp-config.php") && is_writable(dirname(ABSPATH) . "/wp-config.php")) {
+            wp_config_delete('/');
         } else {
             print return_error();
             exit;
         }
 
-        delete_option( "idpay_cf7_options" );
-        delete_option( "idpay_cf7_version" );
+        delete_option("idpay_cf7_options");
+        delete_option("idpay_cf7_version");
     }
 
-    public static function update() {
+    public static function update()
+    {
         global $wpdb;
         $table_name = $wpdb->prefix . "cf7_transactions";
-        $version = get_option( 'idpay_cf7_version', '1.0' );
+        $version = get_option('idpay_cf7_version', '1.0');
 
-        if ( version_compare( $version, '2.1.1' ) < 0 ) {
+        if (version_compare($version, '2.1.1') < 0) {
```

Comment:
you didn't add another else so you won't create your new table for those who are already using the plugin

@MimDeveloperTv